packages/blue-sdk/src/addresses.ts

```diff
@@ -59,6 +59,9 @@ export interface ChainAddresses {
   adaptiveCurveIrm: Address;
   publicAllocator?: Address;
   metaMorphoFactory?: Address;
+  vaultV2Factory?: Address;
+  vaultV1AdapterFactory?: Address;
+  morphoRegistry?: Address;
```

Comment:
```suggestion
  morphoVaultV1AdapterFactory?: Address;
  registryList?: Address;
```

Just to keep exact onchain naming